projects/novo-elements/src/elements/form/FieldInteractionApi.ts

```diff
@@ -568,6 +567,9 @@ export class FieldInteractionApi {
             });
           },
         });
+        if (config.hasOwnProperty('format')) {
+          newConfig.format = config.format;
+        }
```

Comment:
Do we need to grab format from the existing control if one is not passed into modifyPickerConfig?